Clinical-Genomics/scout
# Delete variants by analysis type

- Allow to delete variants for cases with one or more analysis type in "wes", "wgs" and "panel" with "wgs" as a default.
- Print also case status and if it's research in the log resuly

**How to test**:
1. Have cancer case and demo case on a local instance of scout
1. See the difference in results when you remove variants with the 2 commands
  - `scout --loglevel WARNING --demo delete variants -u clark.kent@mail.com --analysis-type wes  --analysis-type panel --status inactive --rank-threshold 5 --dry-run`
  - `scout --loglevel WARNING --demo delete variants -u clark.kent@mail.com --analysis-type wes   --status inactive --rank-threshold 5 --dry-run`


**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
